app/controllers/teams_controller.rb

```diff
@@ -53,7 +53,7 @@ def new
   # GET /teams/1/edit
   def edit
     @team = current_team
-    @user_teams = current_user.teams.order(:name)
+    @user_teams = current_user.teams.not_archived.order(:name)
```

Comment:
I've seen a lot of `.order(:name)` laying around the code for `Team`, should we maybe cache into its on scope?

```ruby
class Team < ActiveRecord::Base
  ...
  scope :by_name, -> { order(:name) }
  ...
end
```
:nail_care:

Reply:
Sure! Thank you for the tip.